spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
```

Comment:
Actually, it isn't. TCP port is freely configurable through Pyload WebUI, port 5080 is just a predefined (although different than pyloads default port 8001). That reminds me complication with "pyload.sc" file.